extra/supervisord/celeryd.conf

```diff
@@ -24,10 +24,8 @@ startsecs=10
 ; Increase this if you have very long running tasks.
 stopwaitsecs = 600
 
-; When resorting to send SIGKILL to the program to terminate it
-; send SIGKILL to its whole process group instead,
-; taking care of its children as well.
-killasgroup=true
+; Causes supervisor to send the stop signal to the whole process group.
```

Comment:
It is not in the documentation, but if we take a look at the `test_stopasgroup` function in the supervisor [sources](https://github.com/Supervisor/supervisor/blob/master/supervisor/tests/test_process.py), we can see that SIGTERM signal is used.